README.md

```diff
@@ -1,54 +1,36 @@
 # React Native Test App
 
+[![Open in Visual Studio Code](https://open.vscode.dev/badges/open-in-vscode.svg)](https://open.vscode.dev/microsoft/react-native-test-app)
 ![build](https://github.com/microsoft/react-native-test-app/workflows/build/badge.svg)
-[![npm version](https://badgen.net/npm/v/react-native-test-app)](https://www.npmjs.com/package/react-native-test-app)
-
-React Native Test App provides test apps for all platforms as a package. Bring
-your own JS bundle, and let us manage the native bits.
-
-This is a work in progress. You can read and discuss the RFC at
-[React Native: Discussions and Proposals](https://github.com/react-native-community/discussions-and-proposals/pull/204).
-
-For more details, please refer to the
-[wiki](https://github.com/microsoft/react-native-test-app/wiki).
-
-## Dependencies
-
-`react-native-test-app` is published with source code only. You will still need
-to compile the test apps yourself.
-
-- **Android**:
-  - [Android Studio](https://developer.android.com/studio) 3.6 or later
-    - Android SDK Platform 29
-    - Android SDK Build-Tools 29.0.3
-    - To install the required SDKs, go into **Preferences** ❭ **Appearance &
-      Behavior** ❭ **System Settings** ❭ **Android SDK**.
-- **iOS/macOS**:
-  - [Xcode](https://apps.apple.com/app/xcode/id497799835?mt=12) 11.3 or later
-  - [CocoaPods](https://cocoapods.org/)
-- **Windows**:
-  - Ensure that
-    [Developer Mode](https://docs.microsoft.com/en-us/windows/uwp/get-started/enable-your-device-for-development)
-    is turned on in Windows Settings app
-  - Install development dependencies as described in the
-    [React Native for Windows documentation](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies)
-  - [Google Chrome](https://www.google.com/chrome/) (optional, but recommended
-    for JS debugging)
-
-# Quick Start
+[![npm version](https://img.shields.io/npm/v/react-native-test-app)](https://www.npmjs.com/package/react-native-test-app)
+
+- [Quick Start](#quick-start)
+- [Configuring the Test App](#configuring-the-test-app)
+- [Known Issues](#known-issues)
```

Comment:
super minor, feel free to ignore: this is not needed technically because GH has now added a table of content feature to READMEs

Reply:
I feel it's less discoverable so I'm going to leave it there for now.